pages/concepts/protocols/portal-sub-protocols/beacon-chain.md

```diff
@@ -0,0 +1,49 @@
+# Beacon Network
+
+***Note that this network is not yet fully production-ready***
```

Comment:
```suggestion
***Note that this network is not production-ready***
```

I think we can be more direct here. Core specs are still changing, so it's not even close from a user point of view.